Core/Src/dti.c

```diff
@@ -64,11 +63,35 @@ void dti_set_torque(int16_t torque)
 
 	int16_t ac_current = (((float)average / EMRAX_KT) * 10); /* times 10 */
 
-	//serial_print("Commanded Current: %d \r\n", ac_current);
+	// serial_print("Commanded Current: %d \r\n", ac_current);
 
 	dti_set_current(ac_current);
 }
 
+void dti_set_regen(float current_target)
+{
```

Comment:
as per my last comment, id much much prefer we multiply by 10 before passing into this function, then handling int16s here. Not a huge deal, but is there any obvious reason why we cant do it that way?

Reply:
I like it better that the * 10 when sending to the MC is abstracted behind the send value function.

Reply:
just froma  pure performance perspective, handling floats in code is less efficient. While indiivdual instances arent gonna be a big deal, handling these all the same way is important, and if we accumulate a lot of floats it could be problematic

Reply:
but def ur call, it shouldnt be detrimental